cli/internal/cmd/init.go

```diff
@@ -246,6 +248,21 @@ func (d *initDoer) Do(ctx context.Context) error {
 	return nil
 }
 
+func (d *initDoer) logGRPCStateChanges(ctx context.Context, conn *grpc.ClientConn) {
+	go func() {
```

Comment:
We should wait for the goroutine to finishe after the connection is done.

Reply:
While adding a waitgroup and waiting for the routine to end I noticed that the context we pass into `conn.WaitForStateChange` has to be canceled or the function would block since it doesn't know if `conn` may be reused for other calls.
This is also consistent with the function doc:

> // WaitForStateChange waits until the connectivity.State of ClientConn changes from sourceState or
> // ctx expires. A true value is returned in former case and false in latter.